public/index.html

```diff
@@ -67,6 +74,37 @@
     simulation.force("link")
         .links(graph.links);
 
+    var requestsUsedEl = document.getElementById("requests_used");
+    requestsUsedEl.textContent = graph.requestsUsed;
+
+    var nodesNumberEl = document.getElementById("nodes_number");
+    nodesNumberEl.textContent = graph.nodes.length;
+
+    var lastUpdateEl = document.getElementById("last_update");
+    lastUpdateEl.textContent = graph.lastUpdate;
+
+    var refreshIntervalEl = document.getElementById("refresh_interval");
+
+    window.setInterval(function() {
+      var now = new Date();
+
+      var nextRefresh = new Date(graph.lastUpdate);
+      nextRefresh.setSeconds(nextRefresh.getSeconds() + graph.refreshInterval);
+
+      var currentWaitTime = refreshIntervalEl.textContent;
+
+      if (currentWaitTime == "") {
+        refreshIntervalEl.textContent = Math.floor((nextRefresh - now) / 1000);
+        return; 
+      }
+      
+      if (currentWaitTime == 0) {
+        refreshIntervalEl.textContent = "Refreshing..."
```

Comment:
Maybe the page could actually refresh itself at the end of the counter? It is already done somewhere under here, but it's on a fixed 60s interval.

It would also be nice to refresh the data of the D3 graph without actually reloading the whole page, but that's another story.